### PR TITLE
added dao classes

### PR DESCRIPTION
**Pull request by Rakesh27git**

### CODE REVIEW

**Review by amith-dd — APPROVED** [review]

File: src/main/java/com/ty/workSpace_Management/dao/WorkSpaceEntityDao.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		if (floor.isPresent()) {
+			return floor.get();
+		}
+		return null;
+	}
+
+	public WorkSpaceEntity updateWorkSpaceEntity(String workSpaceId, WorkSpaceEntity workSpaceEntity) {
+		WorkSpaceEntity entity = getWorkSpaceEntity(workSpaceId);
+		if (entity != null) {
+			workSpaceEntity.setWorkSpaceId(workSpaceId);
```
Comment (bhaskarradha123) [inline src/main/java/com/ty/workSpace_Management/dao/WorkSpaceEntityDao.java:35]: try a code for checking capacity availability cost and type . check the object values if it  wont have the value try to set the old values

File: src/main/java/com/ty/workSpace_Management/dao/FloorDao.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		if (floor.isPresent()) {
+			return floor.get();
+		}
+		return null;
+	}
+
+	public FloorEntity updateFloorEntity(String floorId, FloorEntity floorEntity) {
+		FloorEntity floor = getFloor(floorId);
+		if (floor != null) {
+			floorEntity.setFloorId(floorId);
```
Comment (bhaskarradha123) [inline src/main/java/com/ty/workSpace_Management/dao/FloorDao.java:34]: pls write a code in such a way that check the user were providing the workspaces value ..if wont try to set old value

File: src/main/java/com/ty/workSpace_Management/repo/WorkSpaceRepo.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.ty.workSpace_Management.entity.WorkSpaceEntity;
+
+public interface WorkSpaceRepo extends JpaRepository<WorkSpaceEntity, String> {
+
+	@Query("Select w From WorkSpaceEntity w where w.workSpaceCapacity=?1")
+	public WorkSpaceEntity getByCapacity(int workSpaceCapacity);
```
Comment (bhaskarradha123) [inline src/main/java/com/ty/workSpace_Management/repo/WorkSpaceRepo.java:11]: change the code 

return List of workspaceentity based on capacity

File: src/main/java/com/ty/workSpace_Management/dao/FloorDao.java (hunk 2 above)
Comment (Rakesh27git) [reply src/main/java/com/ty/workSpace_Management/dao/FloorDao.java:34]: i will be  doing this validation in the service layer

File: src/main/java/com/ty/workSpace_Management/repo/WorkSpaceRepo.java (hunk 3 above)
Comment (Rakesh27git) [reply src/main/java/com/ty/workSpace_Management/repo/WorkSpaceRepo.java:11]: ill be doing this validation in the service layer

**Review by bhaskarradha123 — APPROVED** [review]

**Pull request merged**